config/nginx.conf

```diff
@@ -0,0 +1,55 @@
+worker_processes 1;
+
+events {
+  worker_connections 1024;
+}
+
+http {
+  upstream core {
+    server core:8080 max_fails=3 fail_timeout=60s;
+  }
+
+  gzip on; # Enables compression, incl Web API content-types
+  gzip_types
+    "application/json;charset=utf-8" application/json
+    "application/javascript;charset=utf-8" application/javascript text/javascript
+    "application/xml;charset=utf-8" application/xml text/xml
+    "text/css;charset=utf-8" text/css
+    "text/plain;charset=utf-8" text/plain;
+
+  server {
+    listen 80;
+    port_in_redirect off;
+
+    root /data/apps;
+
+    client_max_body_size 10m;
```

Comment:
We usually perform requests with payloads bigger than 10 MiB (of course much better than the 1MiB default value), I'm not sure what value has the WHO servers but could easily hit ```413 Request Entity too large``` errors.

To avoid setting it to ```0``` (no limit) I have ```100M``` for my docker instances.

Reply:
We are using the defaults from dhis2/docker-compose, but yes, we should use more tolerant values. Will do.